From b7f7a583e8a63b1787c3a98f4c43ccbb6c3e39df Mon Sep 17 00:00:00 2001
From: babak sarashki <babak.sarashki@windriver.com>
Date: Sun, 3 Nov 2019 14:36:48 -0800
Subject: [PATCH 06/20] openldap ldaprc currentdir

From Stx 1901: openldap-ldaprc-currentdir.patch

Disables opening of ldaprc file in current directory.
Disables opening of ldaprc file in current directory.

Resolves: #38402
Upstream: ITS #1131
Author: Henning Schmiedehausen <hps@intermeta.de>
---
 libraries/libldap/init.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/libraries/libldap/init.c b/libraries/libldap/init.c
index 9b877a9..89235f9 100644
--- a/libraries/libldap/init.c
+++ b/libraries/libldap/init.c
@@ -352,9 +352,6 @@ static void openldap_ldap_init_w_userconf(const char *file)
 	if(path != NULL) {
 		LDAP_FREE(path);
 	}
-
-	/* try file */
-	openldap_ldap_init_w_conf(file, 1);
 }
 
 static void openldap_ldap_init_w_env(
-- 
2.17.1

